react/SlideToggle/SlideToggle.less

```diff
@@ -0,0 +1,82 @@
+@import (reference) "~seek-style-guide/theme";
+
+.root {
+  display: inline-flex;
+  line-height: @grid-row-height * 3;
+  height: @grid-row-height * 3;
+  font-size: 14px;
+}
+
+.switch {
+  display: inline-block;
+  position: relative;
+  cursor: pointer;
+}
+
+.input {
+  position: absolute;
+  left: 10px;
+  top: 5px;
+  margin: 0;
+
+  &:focus + .slider {
+    box-shadow: 0px 0px 10px 0px @sk-focus;
+  }
+
+  &:active + .slider .slideButton {
+    transform: scale(1.15, 0.85);
```

Comment:
I took the idea from this toggle https://gimme-houston.cloud.seek.com.au/.  I'm happy to go with the majority on whether the animation adds anything.  @stephaniemoss do you have an opinion on this?  Demo page is here http://seek-style-guide--19ad2ab1d6e5f968ef0ae273a6e517939b293e79.surge.sh/slide-toggle/